src/main.rs

```diff
@@ -134,13 +203,13 @@ async fn main() -> anyhow::Result<()> {
         batch_producer_stats.timelines_checked()
     );
 
-    if node_kind.trim() != "pageserver" {
+    if !matches!(node_kind, NodeKind::Pageserver) {
```

Comment:
I think it would be nicer here to have NodeKind impl eq and have a `!=` here.

Reply:
3982d761789c74321fb26344fb823d6a3994e722